packages/contracts/contracts/DependencyRegistryV0.sol

```diff
@@ -114,39 +129,60 @@ contract DependencyRegistryV0 is
         _transferOwnership(_adminACLContract);
     }
 
+    /**
+     * @notice Adds a new license type that can be used by dependencies.
+     */
+    function addLicenseType(bytes32 _licenseType) external {
```

Comment:
I think it makes sense to only provide the ability to add new license types. Just wanted to double click and ensure you agree!

Reply:
++

Reply:
sounds good will close this comment